stdlib/importlib/metadata/__init__.pyi

```diff
@@ -61,7 +61,56 @@ class EntryPoint(_EntryPointBase):
         def attr(self) -> str: ...
     if sys.version_info >= (3, 10):
         dist: ClassVar[Distribution | None]
-        def matches(self, **params: Any) -> bool: ...  # undocumented
+        def matches(
+            self,
+            *,
+            name: str = ...,
+            value: str = ...,
+            group: str = ...,
+            module: str = ...,
+            attr: str = ...,
+            extras: list[str] = ...,
```

Comment:
Does it make sense to include `extras`? The property always returns a newly constructed list, so it can never match as `matches()` only does a shallow `==`. (The same is true for other methods below.)

Reply:
Why? Isn't `[1, 2, 3] == [1, 2, 3]` (for example) true in Python?

Reply:
You are right, I was in JavaScript-mode.